node/lib/util/write_repo_ast_util.js

```diff
@@ -399,6 +400,24 @@ git checkout-index -a -f
         }
     }
 
+    // set up a rebase if there is one
+
+    if (null !== ast.rebase) {
+        const rebase = ast.rebase;
+        const originalSha = commitMap[rebase.originalHead];
+        const ontoSha = commitMap[rebase.onto];
+        const original = yield NodeGit.AnnotatedCommit.lookup(repo,
+                                                              originalSha);
+        const onto = yield NodeGit.AnnotatedCommit.lookup(repo, ontoSha);
+        yield NodeGit.Rebase.init(repo, original, onto, null, null);
```

Comment:
I agree, but I think the documentation and/or contract for `NodeGit.Rebase.init` is the source of confusion. The names of the files for the original and onto commits in rebase-apply/merge file are `origHead` and `onto`, respectively.